charts/ecosystem/values.yaml

```diff
@@ -616,3 +616,28 @@ log4j2Properties: |
 
   logger.ftp.name = ftp
   logger.ftp.level = info
+
+# The name of an existing ConfigMap in the same Kubernetes namespace that the Helm chart will be installed
+# in containing one or more custom Log4j JSON template layouts to customize the Log4j events generated
+# using the 'JsonTemplateLayout' layout.
+#
+# To create a ConfigMap called 'my-json-layouts' with JSON data from existing .json files, run:
```

Comment:
Above, in the readme, you used the singular name `my-json-layout` why the difference ?

Reply:
No reason, just missed out the 's' in the readme :)

Fixed in https://github.com/galasa-dev/helm/pull/99/commits/5c3c928455520b651236009dd0e4c4c8470582ef